vimeo-networking/src/main/java/com/vimeo/networking/model/Interaction.java

```diff
@@ -46,6 +46,7 @@
     private static final String STREAM_AVAILABLE = "available";
     private static final String STREAM_UNAVAILABLE = "unavailable";
 
+    @UseStag
```

Comment:
Done, it doesn't make sense to do this for all enums as many are not parsed directly during serialization, but I added a few UseStag annotations and added tests for all enums: https://github.com/vimeo/vimeo-networking-java/pull/202/commits/c9889e6cc86e2977cbc2f594d0cf1cea0f08acac